app/js/elements/fxos-customizer.js

```diff
@@ -1,12 +1,12 @@
 (function(window) {
+/*jshint maxlen:false*/
```

Comment:
We should remove this and break up the offending code into multiple lines. It's not quite as pretty, but it's better than aborting linting unnecessarily.